ProteGO/UI/Registration/RegistrationSteps/SendCode/ViewModel/RegistrationSendCodeViewModel.swift

```diff
@@ -17,7 +17,14 @@ final class RegistrationSendCodeViewModel: RegistrationSendCodeViewModelType {
 
     func bind(view: RegistrationSendCodeView) {
         view.sendCodeButtonTapEvent.subscribe(onNext: { [weak self] _ in
```

Comment:
tutaj byłoby wtedy 
```
view.sendCodeButtonTapEvent
       .map { view.phoneNumber }
       .validate(type: .phoneNumber)
       .subscribe(onNext: { [weak self] _ in
             self?.model.registerDevice(phoneNumber: view.phoneNumber)
       }, onError: { 
             print(error)
                 // TODO: - Handle Error
       }).disposed(by: disposeBag)

Reply:
Dałbym kilka razy `cmd + z` i miałbym to samo :) W Twoim kodzie cały chain umrze po pierwszym errorze